.gitignore

```diff
@@ -37,5 +37,6 @@ WPI_Native_Libraries.properties
 
 .settings/
 .classpath
+.factorypath
```

Comment:
is this from the eclipse plugin? consider adding a comment

Reply:
Yes all of these three are from the eclipse build tool